### PR TITLE
fix : The ingress backend is modified several times, resulting in residual update events

### PR DESCRIPTION
**Pull request by hahayyum**
After updating k8s ingress(backend) multiple times, the EventUpdate event always exists and cannot end #1027

### CODE REVIEW

File: pkg/kube/translation/ingress.go [hunk 1]
```diff
@@ -273,6 +281,29 @@ func (t *translator) translateIngressV1beta1(ing *networkingv1beta1.Ingress) (*T
 	return ctx, nil
 }
 
+func (t *translator) translateDefaultUpstreamFromIngressV1(namespace string, backend *networkingv1.IngressServiceBackend) *apisixv1.Upstream {
+	var portNumber int32
+	if backend.Port.Name != "" {
+		svc, err := t.ServiceLister.Services(namespace).Get(backend.Name)
+		if err != nil {
+			for _, port := range svc.Spec.Ports {
```
[the remark this answers is not in the document]
Comment (hahayyum) [reply pkg/kube/translation/ingress.go:289]: You're right. I missed this part of the test

**Comment by hahayyum** [comment]
@tokers 
Yes, don't check the old ingress, just keep the latest one

**Comment by hahayyum** [comment]
@tokers 
E2E test (suite features), causing PR operation failure
error info :   localhost: port connection failed, resulting in the failure of normal list / watch
So, what do I need to change？

**Comment by tao12345666333** [comment]
re-run all jobs.

**Comment by tao12345666333** [comment]
@hahayyum sorry for delay, could you please resovle the conflicts? so that we can prepare to merge it

**Comment by hahayyum** [comment]
@tao12345666333 The conflict has been resolved.

**Comment by tao12345666333** [comment]
thanks! Let me take a look

**Review by tao12345666333 — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**